ProteinCartography/esmfold_apiquery.py

```diff
@@ -53,7 +54,7 @@ def post_esmfold_apiquery(fasta: str):
     """
     result = session_with_retry().post(
```

Comment:
Since `verify=False` is a bit of a security concern, I think it'd be good to add a comment here to explain that  this is (hopefully) a temporary fix and will be removed if/when the ESM folks renew their certs.